src/pages/InfoPage.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {useState} from "react";
+import {
+  View,
+  TextInput,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  KeyboardAvoidingView,
+  Platform, FlatList, Image
+} from "react-native";
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import {LinearGradient} from "expo-linear-gradient";
+import {DATA, isMultiSizeItem} from "./SelectionPage";
+
+const renderItem = ({ item }) => {
+  let totalEmission;
+  if (isMultiSizeItem(item)) {
+    const totalEmissionsArray = item.items.map(subItem =>
+        subItem.total_emission
+    );
+    const totalEmissionsSum = totalEmissionsArray.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
+    const averageEmission = totalEmissionsSum / item.items.length;
+    totalEmission = averageEmission.toFixed(3) + " kg CO2e (AVG)";
+  } else {
+    totalEmission = (
+        item.total_emission + " CO2e");
+  }
+
+  return (
+      <View style={styles.itemContainer}>
+        <Image source={item.picture} style={styles.itemImage}/>
+        <View style={styles.itemTexts}>
+          <Text style={styles.itemTitle}>{item.title}</Text>
+          <Text style={styles.itemEmission}>{totalEmission}</Text>
+        </View>
+      </View>
+  );
+};
+
+
+const InfoPage = () => {
+  const [inputText, setInputText] = useState("");
+
+  return (
+      <LinearGradient colors={["lightblue", "white"]} style={styles.container}>
+        <View style={styles.container}>
+          <Text style={styles.title}>List of Items</Text>
```

Comment:
```suggestion
          <Text style={styles.title}>Daily Carbon Emissions Tracker for Medical Devices</Text>
```

Or something along those lines.

Reply:
Okay yea that works!